libs/src/api/account.js

```diff
@@ -135,6 +136,12 @@ class Account extends Base {
         }
       }
 
+      // Create a wAudio user bank address
+      if (createWAudioUserBank && this.solanaWeb3Manager) {
```

Comment:
Can we add a phase for this? Would be very useful imo

Reply:
gooooood call